content/redirects.json

```diff
@@ -77,6 +77,7 @@
   "/data/match-apis": "/data/match-apis/name-match/overview",
   "/data/match-apis/name-match": "/data/match-apis/name-match/overview",
   "/data/account-aggregator": "/data/account-aggregator/overview",
+  "/data/account-aggregator/data/account-aggregator/fi-data-types": "/data/account-aggregator/fi-data-types",
```

Comment:
@riyasuntwal Any reason why this redirect was added?

Reply:
hey, yes, this was giving 404, so we redirected it to a. valid page. Please find the screenshots attached regarding the same.
<img width="1410" alt="image" src="https://github.com/user-attachments/assets/b39ad8a8-7861-474a-9111-e228166daa82">
<img width="1407" alt="image" src="https://github.com/user-attachments/assets/e317892d-0903-45b8-9d8f-e7265e2a1aa0">

Reply:
Got it, I pushed a commit that fixes the original issue which is the path being wrong. Please refer to the changes

[Ref 1](https://github.com/SetuHQ/docs/pull/214/files#diff-0f3884520514c1e89be7f03662bc69c6135596de1ebca99bb742746ff336dd91R22)

[Ref 2](https://github.com/SetuHQ/docs/pull/214/files#diff-ecf9f1556aec932b34b541c9c9894c3206c44a793893b5bbc86fedabde95a6ddR30)